### PR TITLE
chore: remove deprecated `jx console` cmd

### LINKED ISSUE #6928: Remove deprecated console command
### Summary 

The CLI command deprecation list [here](https://jenkins-x.io/commands/deprecation/), lists the `jx console` command as deprecated. But we still support it within Jenkins X despite passing the deprecation date.

We should deprecate remove this command as it's been superseded by `jx ui`.

### PR DESCRIPTION
**Pull request by mrageh**
#### Submitter checklist

- [x] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Description
Remove deprecated `jx console` command.

#### Which issue this PR fixes
fixes #6928

### CODE REVIEW

File: pkg/cmd/console.go [hunk 1]
```diff
@@ -30,44 +26,6 @@ const (
...
-			options.Args = args
-			err := options.Run()
-			helper.CheckErr(err)
-		},
-	}
-	options.addConsoleFlags(cmd)
-	return cmd
-}
-
 func (o *ConsoleOptions) addConsoleFlags(cmd *cobra.Command) {
```
Comment (mrageh) [inline pkg/cmd/console.go]: The remaining functions are used in the `jx open` command, maybe the logic here should be moved into that file?

Comment (mrageh) [reply pkg/cmd/console.go]: By that file I mean https://github.com/jenkins-x/jx/blob/master/pkg/cmd/open.go

**Comment by hferentschik** [comment]
@mrageh, you probably need to run `make fmt`:

```
pkg/cmd/console.go:4: File is not `goimports`-ed (goimports)
	"fmt"
```

**Comment by hferentschik** [comment]
@mrageh, are you going to make the changes we discussed?

**Pull request closed without merging**